### ISSUE TITLE
Tile & fluid ticks "t" grow up very fast while player killed and idling with death screen

### ISSUE THREAD

**Issue by i0xHeX** [PaperMC/Paper#3307]
### Steps/models to reproduce:

- Make sure place to test is not withing spawn range (may be not necessary)
- Make two observers pointing at each other (observer clock)
- Kill yourself, wait for 20-30 seconds being in death screen
- Click "Respawn"
- Teleport back to place where you placed observers and died

### What behaviour is expected:
Observers continue ticking

### What behaviour is observed:
Observers do not tick. You can break and place again - nothing happens. 
Looking into region file, we get this ("t" is very high, but should be 0):
![image](https://user-images.githubusercontent.com/33582973/81240812-25213b00-9011-11ea-8877-61f4618e49a4.png)

### Plugin list:
Auth, CoreProtect, EasyBackup, Harbor, HideLog, LuckPerms, MadhouseMusic, MicroTools, PlugMan, ProtocolLib, RFChairs, ServerRestart, SkinsRestorer, spark, Spartan, WhitelistPlayers*

### Paper build number:
git-Paper-255 (MC: 1.15.2)

### Anything else:
The same happens with fluid ticks. So if some block with fluid got too many high "t", it will produce water every time we update it with another water source and then remove this source.

**Comment by Prof-Bloodstone** [comment]
Despite trying multiple times, I was unable to reproduce it. Neither on build 254, nor 259 - fresh server, stock settings.

As [mentioned on Discord](https://discordapp.com/channels/289587909051416579/289587909051416579/707716097255669762), we had it happen half a year ago. The only plugins we seem to have in common are CoreProtect and LuckPerms.

My guess is that maybe CoreProtect/some config setting causes it to surface.

[According to this Aikar message](https://discordapp.com/channels/289587909051416579/289587909051416579/707721778574786580), Spottedleaf was fixing stuff like this - leaving it here for more details. As mentioned, if trying to reproduce, please use `-Dpaper.ticklist-warn-on-excessive-delay=true -jar paper.jar`.

**Comment by Prof-Bloodstone** [comment]
For reference, I've tried the matrix of following tests:
- Alone or with second account in a different part of the world
- With and without CoreProtect
- Stock settings / with my production spigot.yml config / with paper.yml config
- waiting 20 / 40 / 60 seconds before respawning
- 3 completely different places in the world

This was all done on a fresh world. While inspecting the TileTick when dead, the `t` value got do `-5` and stayed there.

i0xHeX mentioned that on a fresh world, and with all production plugins, he was unable to reproduce, but got `t` to be more negative than I did.

I think that the only differences are that it's fresh world and that I was hosting server locally, while HeX is doing tests on remote VMs.

**Comment by i0xHeX** [comment]
I also can't reproduce that on fresh server with new seed, even with region file and same chunk from production server. I got `t` super low only once, and could not repeat that again in fresh server. Future times I got about -4 or -5, making snapshot while in death screen or after chunk unload.

For now I'm investigating that issue on production server without plugins. Before I tried to put all plugins from production to new server, but issue did not occur. For now after removing all plugins and leave only whitelist - issue does not occur on production. Currently trying different plugins to get the one who related to that. 

But without plugins and both on fresh server I found a bit another issue. The observers may (or may not) tick faster after death screen. After you remove and place again - they become tick again as should. On video I placed two new observers below to show the difference in ticking speed:
https://youtu.be/uwTz2ZyqXx4

**Comment by theminecoder** [comment]
Just a random theory which may or may not have any substance whatsoever, but is your randomTickSpeed gamerule any different on the setup your having issues with?

**Comment by i0xHeX** [comment]
> Just a random theory which may or may not have any substance whatsoever, but is your randomTickSpeed gamerule any different on the setup your having issues with?

No, I didn't change randomTickSpeed gamerule on any server, it's now by default 3.

**Comment by i0xHeX** [comment]
I found reproduce steps are unstable. With few times I tried to do that, with seconds around 15-70 being in death screen - I got the issue only once, with same plugins I first detected it. And still can't detect it again on production server O_o

About flag `-Dpaper.ticklist-warn-on-excessive-delay=true`. I tried it on different server and no warnings were while testing (and no issues were except too low "t" and faster tick speed).

**Comment by Prof-Bloodstone** [comment]
I don't know whether this "after death" issue is known and expected - but in general observers can have different speeds depends on how they were ticked.
I think easiest example is pushing them with pistons.

Place same amount of items in droppers and the complete the circuit by placing observer on gold block - despite starting later, the dropper marked with a diamond will empty first.
![2020-05-07_13 31 54](https://user-images.githubusercontent.com/46570876/81289768-3c8b1300-9067-11ea-91c9-cdf9a4e58483.png)

**Comment by i0xHeX** [comment]
So I spent about 3 hours figuring this out and found that on Spartan plugin presence issue can be reproduced on my production server. I found that in perfect timings (after ~21 seconds after death and ~1-5 seconds to teleport back) this issue is reproducible in most cases. I tested 8 times with Spartan and without. In first case I got 6 times reproducing issue, in second case - 0.

However, I moved that plugin to another clean server, with configs, and couldn't reproduce the issue from 5 attempts, with and without Leaf's flag. So this is remains unclear what's more causing the issue.

**Comment by i0xHeX** [comment]
Recently I made a little hack to force enable on production that flag to warn on excessive delay by modifying bytecode in PaperTickList, and got a lot of warning stack traces after killing myself (selected 3, can provide all if needed):
https://pastebin.com/gbHHJFNq
